src/frontend/src/flows/recovery/displaySeedPhrase.ts

```diff
@@ -1,9 +1,52 @@
-import { html, render } from "lit-html";
+import { html } from "lit-html";
+import { withRef, renderPage } from "../../utils/lit-html";
+import { ref, createRef, Ref } from "lit-html/directives/ref.js";
 import { checkmarkIcon, copyIcon } from "../../components/icons";
 import { mainWindow } from "../../components/mainWindow";
 import { toast } from "../../components/toast";
 
-const pageContent = (seedPhrase: string) => {
+const displaySeedPhraseTemplate = ({
+  seedPhrase,
+  onContinue,
+  copyPhrase: copyPhrase_,
+}: {
+  seedPhrase: string;
+  copyPhrase: () => Promise<void>;
+  onContinue: () => void;
+}) => {
+  const phraseElement: Ref<HTMLElement> = createRef();
+  const phraseCopyElement: Ref<HTMLElement> = createRef();
+
+  const continueButton: Ref<HTMLButtonElement> = createRef();
+  const checkbox: Ref<HTMLInputElement> = createRef();
+
+  // Selects the phrase content to give visual feedback of what's been copied
+  const selectPhrase = () => {
+    withRef(phraseElement, (phraseElement) => {
+      const selection = window.getSelection();
```

Comment:
just in case, a get selection that supports old browser I used in ddg and stylo:

```
export const getSelection = (): Selection | null => {
  if (window && window.getSelection) {
    return window.getSelection();
  } else if (document && document.getSelection) {
    return document.getSelection();
  } else if (document && (document as any).selection) {
    return (document as any).selection.createRange().text;
  }

  return null;
};
```

Reply:
Nice, thanks! I've just reused whatever we were using but I'm making note of this; will add soon but want to test it a bit first.